### PR TITLE
Added the no device error

### PR DESCRIPTION
**Pull request by phistuck**
My computer at work does not have a microphone or a camera plugged in, so I get a false error message. This adds an appropriate error message for this case.

### CODE REVIEW

File: src/js/gumhandler.js [hunk 1]
```diff
@@ -59,8 +61,12 @@ GumHandler.prototype = {
   gotError_: function(error) {
     clearTimeout(this.firstUserCheck_);
     this.gumPendingDialog_.close();
-    this.gumErrorMessage_.innerHTML = error.name;
-    this.gumErrorDialog_.open();
+    if (error.name === 'DevicesNotFoundError') {
+      this.gumNoDeviceDialog_.open();
```
Comment (andresusanopinto) [inline src/js/gumhandler.js]: It is missing a close of dialog whenever the condition stops being true.

Probably is missing:
 1 - close the gumErrorDialog whenever gumNoDeviceDialog is opened.
 2 - close the gumNoDeviceDialog whenever gumErrorDialog is opened.
 3 - close the gumNoDeviceDialog when gotStream succeeds.

**Comment by andresusanopinto** [comment]
@KaptenJansson can you also review? (it lgtm).
 1 - you were the one with the original feature.
 2 - I don't have access to a device with no mic/camera

File: src/index.html [hunk 2]
```diff
@@ -81,6 +81,14 @@
       enabled browser listed at <a href="http://www.webrtc.org">http://www.webrtc.org</a>.</p>
   </paper-dialog>
 
+  <!-- GetUserMedia no audio or video device dialog -->
+  <paper-dialog id="gum-no-device-dialog"
+                heading="Welcome to WebRTC Troubleshooter"
+                backdrop autoCloseDisabled>
+    <p>Your device does not have any audio or video devices connected.
+    Please, plug a microphone or a camera into your device.</p>
```
Comment (KaptenJansson) [inline src/index.html]: Indent by 2 spaces.

I know there is a place above where it''s indented incorrectly (most likely by me;)), will fixed that part of a different PL.

**Comment by KaptenJansson** [comment]
LGTM with nit.

**Comment by andresusanopinto** [comment]
Thanks for the patch @phistuck

**Comment by KaptenJansson** [comment]
Yeah, thanks a lot!

**Pull request merged**